.github/workflows/main.yml

```diff
@@ -217,8 +249,8 @@ jobs:
         shell: bash -l {0}
         run: |
           source $HOME/.cargo/env
-          make SUBSTRATE_PARSER_FULL=1
-          mv ./app/pkg/installer_s.sh ./app/pkg/installer_nanos_xl.sh
+          make SUBSTRATE_PARSER_FULL=1 DISABLE_PREVIOUS=1
+          mv ./app/pkg/installer_s.sh ./app/pkg/installer_nanos_xl_current.sh
       - name: Set tag
         id: nanos_xl
         run: echo ::set-output name=tag_name::$(./app/pkg/installer_nanos_xl.sh version)
```

Comment:
installer_nanos_xl_current.sh should be used

Reply:
ups, done!